cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -269,11 +285,29 @@ def upgrade_5_0_0(session_key, logger):
         "Linux - Group Added/Updated/Deleted"
     ]
 
-    for alert in alerts_to_disable:
-        try:
-            conf_manager.update_savedsearch(alert, {"disabled": 1})
-        except Exception as e:
-            logger.info("Not able to disable the alert={} error={}".format(alert, str(e)))
+    disable_the_alert(conf_manager, logger, alerts_to_disable)
+
+
+def upgrade_5_2_0(session_key, logger):
```

Comment:
Let's create 3 functions:

Function-1 -> handle_newly_added_alerts(<list of tupple(alert-name, macro-name)>)
Function-2 -> handle_removed_alerts(<list of tupple(alert-name, macro-name)>)
Function-3 -> handle_renamed_alerts(<list of tupple(old-alert-name, old-alert-macro, new-alert-name, new-alert-macro)>